src/js_sync.hpp

```diff
@@ -438,13 +612,6 @@ void SessionClass<T>::get_config(ContextType ctx, ObjectType object, ReturnValue
                                  Value::from_string(ctx, String::from_bson(partition_value_bson)));
         }
 
-        auto conf = session->config();
```

Comment:
@kneth What was the reasoning of removing `error` from the returned sync configuration?